python/semantic_kernel/connectors/memory/azure_cognitive_search/azure_cognitive_search_memory_store.py

```diff
@@ -136,6 +141,7 @@ async def create_collection(
                 name=collection_name.lower(),
                 fields=get_index_schema(self._vector_size),
                 vector_search=vector_search,
+                encryption_key=search_resource_encryption_key,
```

Comment:
could you have a look to see if you can add a unit test for this, that tests both with and without key?

Reply:
Hello @eavanvalkenburg , I have mocked and added the tests.